README.md

```diff
@@ -97,6 +97,8 @@ You can pass the following options to `cmake ..`.
 * `-DUNDEFINED_BEHAVIOR_SANITIZER` -- Build with UndefinedBehaviorSanitizer
 * `-DLINK_PROFILER` -- Link with gperftools (available profiler options are listed [here](https://github.com/gperftools/gperftools))
 
+To clean up the `open-source` and `build` folders from previous build, use `cmake --build . --target clean`
```

Comment:
Also add info on which directory this command should be run from.

Reply:
Apologies for missing it, added it